Gemfile

```diff
@@ -1,4 +1,3 @@
 source 'https://rubygems.org' do
-  gem 'cocoapods', '1.0.0'
-  gem 'activesupport', '~> 4.0'
+  gem 'cocoapods', :git => 'https://github.com/CocoaPods/CocoaPods.git', :ref => '3376d6e'
```

Comment:
because no release version of Cocoapods works with Swift 3 yet. And I didn't want to pin to `master`